MekHQ/src/mekhq/campaign/personnel/Person.java

```diff
@@ -1660,6 +1670,8 @@ public static Person generateInstanceFromXML(Node wn, Campaign c, Version versio
                     advantages = wn2.getTextContent();
                 } else if (wn2.getNodeName().equalsIgnoreCase("edge")) {
                     edge = wn2.getTextContent();
+                } else if (wn2.getNodeName().equalsIgnoreCase("edgeAvailable")) {
+                    retVal.currentEdge = Integer.parseInt(wn2.getTextContent());
```

Comment:
Does `retVal.currentEdge` need to start as -1 and get set to edge at the end if it stays -1? I'm thinking of the case where you load a campaign and the people have not been updated yet.

Reply:
I'm not quite sure I understand the case you're describing.  I want to be able to save and load campaigns where currentEdge has been spent for the week, and that seems to work as I planned in testing.